packages/common/src/i18n/en.ts

```diff
@@ -206,4 +206,13 @@ export default {
   alerts: {
     closeAlerts: "Close Alerts",
   },
+  mainMenu: {
+    resourcesWithinReach: "Resources within reach",
+    about: "About",
+    termsOfUse: "Terms of Use",
+    privacyPolicy: "Privacy Policy",
+    close: "Close",
```

Comment:
I think this could belong in our glossary rather than main menu since it's pretty generic.